### PR TITLE
Update the gcs chunk_size documentation.

### PR DESCRIPTION
**Pull request by cafuego**
Remove `1g` from the examples, as the GCS repository chunk_size can be at most 100m. As illustrated by the error message when trying to set it to `1g`:

```
$ curl -H 'Content-type: application/json' -u user:pass -XPUT http://127.0.0.1:9200/_snapshot/backup -d '{
  "type": "gcs",
  "settings": {
    "bucket": "elk-backup",
    "client": "backups",
    "chunk_size": "1g"
  }
}'
{"error":{"root_cause":[{"type":"repository_exception","reason":"[backup] failed to create repository"}],"type":"repository_exception","reason":"[backup] failed to create repository","caused_by":{"type":"illegal_argument_exception","reason":"failed to parse value [1g] for setting [chunk_size], must be <= [100mb]"}},"status":500}
```

### CODE REVIEW

**Comment by cafuego** [comment]
This is a rewrite of https://github.com/elastic/elasticsearch/pull/38680 with the correct committer email, so that the CLA check passes.

Pinging @original-brownbear

**Comment by original-brownbear** [comment]
@elasticmachine good to test

**Comment by original-brownbear** [comment]
@elasticmachine test this please

**Review by original-brownbear — APPROVED** [review]
LGTM

**Comment by original-brownbear** [comment]
@cafuego thanks1

**Comment by original-brownbear** [comment]
@cafuego the CLA check went back to red? Did you make any changes to your account's email settings?

**Comment by cafuego** [comment]
I did not!

**Comment by original-brownbear** [comment]
@cafuego right you did not :) It looks like our CLA tracker has some issues with GH API rate limits :( Could you rebase your branch against `master` or merge `master` into it to retrigger the PR build? Thanks and sorry for the inconvenience :)

**Comment by cafuego** [comment]
Everything hates me.

**Comment by original-brownbear** [comment]
Jenkins test this

**Comment by original-brownbear** [comment]
@cafuego I just manually looked into the CLA system and couldn't find the email address used for the commit here in there (or any email address for your account for that matter). I remember the build being green once so this is a little confusing to me. Maybe try signing again (and/or) make sure you clicked the verification link in the email you received when signing the CLA?

**Comment by cafuego** [comment]
Ok, have just signed (and confirmed) again.

**Comment by original-brownbear** [comment]
@cafuego sorry we had some trouble with our CLA checker back in Feb. This should be all fixed now. If you're still willing to help you signing the CLA one last time would be much appreciated so I can merge this one in.

Thanks!

**Comment by cafuego** [comment]
Done, however:
- The link to CLA page in the PR check is incorrect, the www.elascisearch.org URL redirects to the www.elasic.co home page instead of the www.elasic.co/contributor-agreement page

**Comment by original-brownbear** [comment]
Jenkins update branch

**Comment by cafuego** [comment]
Re-re-re-re-re-signed the CLA.

**Comment by original-brownbear** [comment]
Jenkins run elasticsearch-ci/docbldesx

@cafuego it worked! will merge once the test failure is green :)

**Comment by original-brownbear** [comment]
@elasticmachine update branch

**Comment by original-brownbear** [comment]
@cafuego it seems that since this branch is so far out of sync with `master` now it won't pass CI. Can you  merge current `master` into it again so I can get green CI and merge it?
Thanks!

**Comment by original-brownbear** [comment]
Jenkins run elasticsearch-ci/docbldesx

**Comment by cafuego** [comment]
Rebase rather than merge, followed by a force-push to overwrite my original `cafuego-gcs-chunk-docs` branch. All green now!

**Comment by original-brownbear** [comment]
Jenkins test this

(gotta run test first :), but thank you so much for your continued patience @cafuego !)

**Comment by original-brownbear** [comment]
@cafuego thanks again, now it's green -> merging :)

**Comment by cafuego** [comment]
🎉  thanks @original-brownbear !

**Pull request merged**
